clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -960,7 +1099,7 @@ bool tools::addOpenMPRuntime(ArgStringList &CmdArgs, const ToolChain &TC,
     CmdArgs.push_back("-Bdynamic");
 
   if (RTKind == Driver::OMPRT_GOMP && GompNeedsRT)
-      CmdArgs.push_back("-lrt");
+    CmdArgs.push_back("-lrt");
```

Comment:
This and the following changes in this file are unrelated and merely formatting fixes. Please, could you reduce the noise and revert these? I am happy for you to submit them as a separate PR.